examples/asr/transcribe_speech.py

```diff
@@ -105,7 +105,9 @@ class TranscriptionConfig:
     pretrained_name: Optional[str] = None  # Name of a pretrained model
     audio_dir: Optional[str] = None  # Path to a directory which contains audio files
     dataset_manifest: Optional[str] = None  # Path to dataset's JSON manifest
-    channel_selector: Optional[int] = None  # Used to select a single channel from multi-channel files
+    channel_selector: Optional[
+        Union[int, str]
```

Comment:
Hmm dunno about this, hydra doesn't support optional union and we don't test this script in CI (or maybe we do with an old QN model). Did you test this locally ?

Reply:
Indeed union works now with the specifics that only primitive types are in the union